tdrs-backend/tdpservice/data_files/serializers.py

```diff
@@ -88,3 +91,79 @@ def validate_file(self, file):
         validate_file_extension(file.name)
         validate_file_infection(file, file.name, user)
         return file
+
+
+def get_xls_serialized_file(data):
```

Comment:
my suggestion is to create a util.py file and move this function to that file instead of serializer file.

Reply:
done :)